startup.py

```diff
@@ -109,7 +109,10 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
                     self.logger.debug("Preparing to launch builtin plugin '%s'" % load_plugin)
                     load_maya_plugins.append(load_plugin)
                     if load_plugin not in maya_module_paths:
-                        maya_module_paths.append(load_plugin)
+                        # Insert at begining of list to give priority to toolkit plugins
```

Comment:
Thanks for prompt review guys. Sorry, I will fix the bad spelling typo and merge. Thank u